debian/clearwater-etcd.postinst

```diff
@@ -96,6 +96,12 @@ case "$1" in
 
         install -D --mode=0644 /usr/share/clearwater/conf/clearwater-etcd.monit /etc/monit/conf.d/
         reload clearwater-monit || /bin/true
+
+        # Restart clearwater-etcd.  Always do this by terminating clearwater-etcd so monit will
+        # restart it more-or-less immediately.  (monit restart seems to have
+        # significant lag.)
+        # Don't fail if it's already stopped.
+        service clearwater-etcd stop || true
```

Comment:
It does (or could) mean something slightly different - `/bin/true` runs a specific binary, while `true` might be interpreted by the shell itself.  However, I'm struggling to see how much difference in behavior you can find between two implementations of the specification "return 0".  Arguably, `true` might be more efficient, but it's going to be negligible.  I'd prefer `true`.